### PR TITLE
Python3.10

### PR DESCRIPTION
**Pull request by chrisJohn404**
Python3.10

### CODE REVIEW

**Comment by chrisJohn404** [comment]
Sorry if this is to big of a pull request.

**Review by sjarman28 — COMMENTED** [review]
Unfortunately, it might be a while until we approve this request. Most of the changes look good, but ljsimpleregisterlookup is still on Python 2.7. If this request were approved, we would not be able to pull new changes from ljm_constants to ljsimpleregisterlookup without it breaking or doing some workaround, so we do not want to approve this request until ljsimpleregisterlookup is upgraded. A test deployment failed when I tried to upgrade ljsimpleregisterlookup to Python 3, and getting that working is a low priority at the moment.

**Comment by chrisJohn404** [comment]
Not ideal, but reverted back to Python2.

File: save_changes.py [hunk 1]
```diff
@@ -6,28 +6,42 @@
 import os
 import subprocess
 import sys
+from os import path
```
Comment (sjarman28) [inline save_changes.py:9]: Please clean up this import. There are some references using `os.path` and `path`. I think we should revert back to just having `import os`, but making the changes to import each function individually would be agreeable as well.

File: validate.py [hunk 2]
```diff
@@ -4,10 +4,24 @@
 """
 import sys
 import ljmmm
+import traceback
+import os
```
Comment (sjarman28) [inline validate.py:8]: Same thing as `save_changes.py`, please clean up the os module import.

**Review by sjarman28 — CHANGES_REQUESTED** [review]
Everything except the two os module imports I commented on look good to me.

About upgrading to Python 3:
I did upgrades in the backend for Python3, bumped the runtime to Python 3.10.4,  and bumped some of the dependencies up to the newest release versions. The updated LJSimpleRegisterLookup works locally, but fails when trying to deploy. It appears to be a CORS header error, so there must be a configuration missing somewhere.

File: validate.py (hunk 2 above)
Comment (chrisJohn404) [reply validate.py:8]: I agree.

**Review by sjarman28 — APPROVED** [review]
Looks good, thanks.

**Pull request merged**